atlasdb-impl-shared/src/test/java/com/palantir/atlasdb/internalschema/TimestampPartitioningMapTest.java

```diff
@@ -33,32 +35,34 @@
 
     @Test
     public void throwsIfInitialMapIsEmpty() {
-        assertThatThrownBy(() -> TimestampPartitioningMap.of(ImmutableRangeMap.of()))
-                .isInstanceOf(IllegalArgumentException.class)
+        assertThatLoggableExceptionThrownBy(() -> TimestampPartitioningMap.of(ImmutableRangeMap.of()))
+                .isInstanceOf(SafeIllegalArgumentException.class)
                 .hasMessageContaining("its span does not cover precisely all timestamps");
```

Comment:
you can actually use `hasLogMessage` and then `hasExactlyArgs`, but this is probably fine _for now_.